co-log-core/src/Colog/Core/Action.hs

```diff
@@ -270,30 +281,62 @@ cmapM :: Monad m => (a -> m b) -> LogAction m b -> LogAction m a
 cmapM f (LogAction action) = LogAction (f >=> action)
 {-# INLINE cmapM #-}
 
--- | @divide@ combinator from @Divisible@ type class.
+{- | @divide@ combinator from @Divisible@ type class.
+
+>>> logString = LogAction putStrLn
+>>> logInt = LogAction print
```

Comment:
Btw, `LogAction print` looks like a good addition to the `Colog.Core.IO` module

Reply:
Should I go ahead and add this in this pr, or open should we open an issue?

Reply:
@cronokirby I think separate issue is better 👍